SPM Example/PortalSwift/ViewController.swift

```diff
@@ -590,26 +733,34 @@ class ViewController: UIViewController, UITextFieldDelegate {
         self.logger.error("Error: Do you have `GDRIVE_CLIENT_ID=$(GDRIVE_CLIENT_ID)` in your Secrets.xcconfig?")
         throw PortalExampleAppError.environmentNotSet()
       }
+      guard let BACKUP_WITH_PORTAL: String = infoDictionary["BACKUP_WITH_PORTAL"] as? String else {
```

Comment:
```suggestion
      guard let backupWithPortal: String = infoDictionary["BACKUP_WITH_PORTAL"] as? String else {
```
Swift has a camelCase convention for variable names